README.md

```diff
@@ -167,6 +167,7 @@
 - 🌍🔧 [GPT-3 Tailwind CSS code generator](http://gpt-tailwind.com/) - OpenAI GPT-3 powered Tailwind CSS code generator.
 - 🌍🔧 [Stitches](https://stitches.hyperyolo.com/) - Template generator with Tailwind (online).
 - 🌍🔧 [tail-animista](https://tail-animista.vercel.app) - Configurable custom animation utilities generator for Tailwind CSS.
+- 🌍🔧 [brands-tail-color](https://brands-tail-color.vercel.app/) - Configuration generator of various brand colors for background, border and text for Tailwind CSS.
```

Comment:
I think this would be clearer, what do you think?

```suggestion
- 🌍🔧 [brands-tail-color](https://brands-tail-color.vercel.app/) - Configuration generator using various brands' colors.
```

Reply:
Sure thing innocenzi, your suggestion sounds more clear